backend/src/main/java/se/chalmers/dat265/group1/storage/repository/GenericRepository.java

```diff
@@ -143,20 +144,23 @@ public T updateObject(T object) {
 
     private T execute(JsonObject jsonObject2, StringBuilder query2) {
         try {
-            PreparedStatement ps = getPreparedStatementFromJSON(jsonObject2, query2); //maps model data to the query
+            PreparedStatement ps = getPreparedStatementFromJSON(jsonObject2, query2); //maps se.chalmers.dat265.group1.model data to the query
             ps.executeUpdate();
             ResultSet rs = ps.getGeneratedKeys();
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Use try-with-resources or close this "ResultSet" in a "finally" clause. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2095)